### ISSUE TITLE
BUG: There's a momentary flicker before entering the Excalidraw interface through links in notes

### ISSUE THREAD

**Issue by sfu2008** [zsviczian/obsidian-excalidraw-plugin#1567]
Hi, I really like this plugin, but there is a question I'd like to give feedback on,

I've encountered a problem when accessing Excalidraw pages through links in notes. There's a momentary flicker before entering the Excalidraw interface, which is particularly jarring against a dark background.

THANKS!

**Comment by zsviczian** [comment]
yes, there is. Excalidraw initiates with the light background the switches to dark. That can cause a flicker.
